app/models/assessment.rb

```diff
@@ -37,21 +42,29 @@ def remarks
     Remarks.new(id)
   end
 
+  def version_3?
+    version == '3'
+  end
+
+  def version_4?
+    version == '4'
+  end
+
   private
 
   def matter_proceeding_type_required?
-    version == '3'
+    version_3?
   end
 
   def proceeding_type_codes_validations
-    return if version == '3'
+    return if version_3?
 
     proceeding_type_codes.each do |code|
       errors.add(:proceeding_type_codes, "invalid: #{code}") unless code.to_sym.in?(ProceedingTypeThreshold.valid_ccms_codes)
     end
   end
 
   def proceeding_types_codes_required?
-    version != '3'
+    version_4?
```

Comment:
Is this too fragile? Will it break on the inevitable v5?  Should we have a `version>3?` method?